### ISSUE TITLE
New Public Domain Books

### ISSUE THREAD

**Issue by StellaAthena** [r-three/licensed-pile#48]
We are going to make a dataset of public domain books that people don't currently know are in the public domain.

Current estimated count: 443,333 books, another 200,000 have incomplete metadata and may end up being included.

GitHub Repo for work: https://github.com/EleutherAI/pd-books

**Comment by craffel** [comment]
@StellaAthena does this overlap with https://huggingface.co/datasets/institutional/institutional-books-1.0 ?

**Comment by baberabb** [comment]
yeah, will have a look any overlap, but they have ~16,000 books with publication date > 1929 (the threshold we care about)
